e2e/scripts/k8sBlockLongHaul.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash -e
+
+test_source_repo=$(pwd)
```

Comment:
if it's going away then i wouldn't worry about it. Relying on `pwd` just has its issues